frontend/src/redux/parameters/reducer.ts

```diff
@@ -79,6 +81,14 @@ const reducer = (state: ParametersState = initialState, action: AnyAction) => {
           [action.payload.projectId]: action.payload.displayedMetrics,
         },
       };
+    case getType(updateAllDisplayedMetrics):
+      return {
+        ...state,
+        displayedMetrics: {
```

Comment:
Could we rename this to `currentDisplayedMetrics` to be consistent with the naming of the other parameters in this Redux state?